binding/flutter/lib/porcupine_error.dart

```diff
@@ -9,49 +9,84 @@
 // specific language governing permissions and limitations under the License.
 //
 
-class PvAudioException implements Exception {
+// ignore_for_file: unnecessary_brace_in_string_interps
+
+class PorcupineException implements Exception {
```

Comment:
Can you check "Error" vs "Exception" in Flutter? I can't remember, but they were opinionated on which word you use when.

Reply:
Exception is the normal exceptions we know, i.e. throw and catch. Errors are not supposed to be catched and probably done by programmer's mistake.